substrate/frame/contracts/src/migration/v11.rs

```diff
@@ -27,7 +27,7 @@ use crate::{
 use sp_runtime::TryRuntimeError;
 
 use codec::{Decode, Encode};
-use frame_support::{pallet_prelude::*, storage_alias, DefaultNoBound};
```

Comment:
Why was this removed? It does clearly not build anymore. Please check the CI output.